tests/journey/test_chamber_of_deputies_official_missions_dataset.py

```diff
@@ -20,7 +20,6 @@ def test_fetch(self):
             'canceled', 'report_status', 'report_details_link'
         ]
         self.assertTrue(np.array_equal(expectedColumns, actualColumns))
-        self.assertEqual(56, len(df))
```

Comment:
I totally agree that this test is a problem _per se_. However just for maintainability sake and for the community to follow the work on the project I'm not in favor of “smuggling” this change in the middle of a PR related to something else (code quality, health etc.).

There is a greater discussion about the [role of tests in the project](https://github.com/datasciencebr/serenata-toolbox/issues/146) going on.

Would you mind reverting this change and let this to another PR/discussion?

BTW your commits are larger than I would recommend: not in terms of amount of changes, but in the purpose of the changes committed:

* ace718d: identified static methods and removed test that doesn't make sense anymore
* d2c4622: fixed spaces according to pep8 e128 and updated minor patch


In the first one listed you committed this change together with another unrelated  change, so you cannot use `git revert` or `git rebase` to fix that. In the second is some other PR gets merged first, you cannot use `git` to revert a version bump… In general my advice is: if your commit message is _X and Y_ you should commit twice: first commit _X_ and then commit _Y_ ; )

Reply:
I do agree. :)

Reply:
I removed the test with the thought that I was removing unused code as part of the project health. But I agree, it was bad and it is not consistent with others changes.

Oh, that's true. The commits are terrible.

I will fix that and come back
Thank you and srry for the trouble, like I told @lipemorais, the project that is contributing to me hahaha